RetailCoder.VBE/Inspections/InspectionsUI.resx

```diff
@@ -555,4 +555,13 @@
   <data name="VariableTypeNotDeclaredInspectionResultFormat" xml:space="preserve">
     <value>{0} '{1}' is implicitly 'Variant'</value>
   </data>
-</root>
+  <data name="MalformedAnnotationInspectionMeta" xml:space="preserve">
+    <value>An annotation comment is malformed.</value>
```

Comment:
"Malformed" is fine wording for internal stuff, but exposing it to users seems a bit pretencious - I'd go with a message that doesn't make it seem like it's the user (or rather, their code) that's at fault here. Something like:

> Comment looks like it could be a Rubberduck annotation, but Rubberduck was not able to parse it as such.